dataframe-openapi-generator/README.md

```diff
@@ -0,0 +1,13 @@
+## :dataframe-openapi
+
+This module, published as `dataframe-openapi-generator` contains all logic and tests for DataFrame to be able to import
+OpenAPI specifications as auto-generated data schemas. This module is a sister module to
+[`dataframe-openapi`](../dataframe-openapi):
+
+- `dataframe-openapi-generator` is used as a dependency of the Gradle plugin and Jupyter plugin to be able to generate
```

Comment:
KSP can import schemas too, right?

Reply:
yes definitely, but in the documentation we don't explicitly name "ksp" to be separate from the gradle plugin, right? Even I am not entirely sure about the difference because the gradle plugin also uses ksp to generate the sources right?